src/jit/_typeinfo.h

```diff
@@ -416,7 +416,6 @@ class typeInfo
     static bool AreEquivalentModuloNativeInt(const typeInfo& verTi, const typeInfo& nodeTi)
     {
         if (AreEquivalent(verTi, nodeTi)) return true;
-        // Otherwise...
```

Comment:
Some comments were removed, like this one, because they are unnecessary and uninformative.